corehq/apps/accounting/tests/test_migrations.py

```diff
@@ -0,0 +1,135 @@
+import random
+from datetime import date, timedelta
+
+from django.test import TransactionTestCase
+
+from corehq.apps.accounting.tests import generator
+from corehq.apps.accounting.models import (
+    BillingAccount,
+    DefaultProductPlan,
+    SoftwarePlanEdition,
+    Subscription,
+)
+from corehq.apps.accounting.tasks import assign_explicit_community_subscriptions
+from corehq.apps.domain.models import Domain
+
+
+class TestExplicitCommunitySubscriptions(TransactionTestCase):
```

Comment:
Are you using `TransactionTestCase` on purpose? Last time I tried to use it it ran the couchdb sync after every test which made it VERY slow.